conf.py

```diff
@@ -15,7 +13,15 @@
 copyright = '2023, Your name'
 author = 'Your Name'
 
-extensions = ['pechakucha.pechakucha']
+extensions = [
+    'Pechakucha.pechakucha_directive',   
+     #"myst_nb",
```

Comment:
if you take this one out, then you have to use restructured text, no markdown.

Reply:
im confused on what you mean by this

Reply:
you commented out the `myst_nb` extension, which was enabled to allow sphinx to understand markdown.  without that, it only understands restructured text.